README.md

```diff
@@ -1 +1,74 @@
-# emporium-ecom
\ No newline at end of file
+# Emporium - Ecommerce store built using Zippy-UI
+
+Pages and what Components are used from Zippy-UI:
+
```

Comment:
Add your live url or code sandbox link here 
Everything else is good

Reply:
Added
Thanks!